hazelcast/near_cache.py

```diff
@@ -193,9 +196,19 @@ def _is_eviction_required(self):
     def get_statistics(self):
         """
         Returns the statistics of the NearCache.
-        :return: (Number, Number), evicted entry count and expired entry count.
+        :return: (Dict), Dictionary that stores statistics related to this near cache.
         """
-        return self._evicted_count, self._expired_count
+        stats = {
+            "creation_time": self._creation_time,
```

Comment:
owned entry memory is cost is size in this case.I will change its name. Persistance related statistics are not collected since near cache preloader is not implemented yet. 
I can't see the invalidation related statistics in the design document or the java doc string. Are they added later ? https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/client/impl/ClientEngine.java#L83